app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -111,8 +111,11 @@ class HomePresenter(
                 view?.showRefreshing(true)
                 val solTokens = userInteractor.loadUserTokensAndUpdateLocal(tokenKeyProvider.publicKey.toPublicKey())
                 async { userInteractor.loadUserRates(solTokens) }
+                async {
+                    val bundles = ethereumInteractor.getListOfEthereumBundleStatuses()
+                    ethereumInteractor.loadWalletTokens(bundles)
```

Comment:
how about making a method loadWalletTokens and make getListOfEthereumBundleStatuses call inside?
it looks like business logic to keep this methods together

Reply:
okay